Sources/Algorithms/Chunked.swift

```diff
@@ -246,3 +246,136 @@ extension Collection {
     try chunked(on: projection, by: ==)
   }
 }
+
+//===----------------------------------------------------------------------===//
+// chunks(of:)
+//===----------------------------------------------------------------------===//
+
+/// A collection that presents the elements of its base collection
+/// in `SubSequence` chunks of any given size.
+///
+/// A ChunkedCollection is a lazy view on the base Collection, but it does not implicitly confer
+/// laziness on algorithms applied to its result.  In other words, for ordinary collections `c`:
+///
+/// * `c.chunks(of: 3)` does not create new storage
+/// * `c.chunks(of: 3).map(f)` maps eagerly and returns a new array
+/// * `c.lazy.chunks(of: 3).map(f)` maps lazily and returns a `LazyMapCollection`
```

Comment:
For this to be true you’ll need to add conditional conformance to `LazyCollectionProtocol` when the base collection conforms.

Reply:
Conformance added